driverconfig/config.yaml

```diff
@@ -40,6 +41,13 @@ drivers:
         external-health-monitor: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.4.0
   - name: unity
     configVersions:
+      - configVersion: v2.5.0
+        useDefaults: true
+        supportedVersions:
+          - version: v121
+          - version: v123
+          - version: v124
+          - version: v125
       - configVersion: v2.4.0
         useDefaults: true
```

Comment:
Please set this to false and add the sidecars accordingly

Reply:
Done